Food_Tracker_App/app/src/main/res/layout/activity_view_recipe.xml

```diff
@@ -0,0 +1,238 @@
+                app:layout_constraintBottom_toTopOf="@id/ingredientHeaders"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/recipeDisplayCategory" />
+
+            <include layout="@layout/ingredient_header_row"/>
+
+            <androidx.recyclerview.widget.RecyclerView
+                android:background="@color/background_light"
+                android:padding="8dp"
+                android:id="@+id/recipeDisplayIngredientList"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toTopOf="@id/recipeDisplayComment"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHeight_min="25dp"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/ingredientHeaders" />
+
+            <LinearLayout
+                android:id="@+id/recipeDisplayComment"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:background="@color/background_light"
+                android:layout_marginVertical="16dp"
+                android:padding="8dp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/recipeDisplayIngredientList" >
+
+                <TextView
+                    style="@style/textBody"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="@string/comments"
+                    android:textStyle="bold"
+                    android:layout_weight="1"/>
+
+                <TextView
+                    android:id="@+id/comments"
+                    style="@style/textBody"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="2"/>
+            </LinearLayout>
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+    </ScrollView>
+
+    <Button
+        android:id="@+id/recipeReturnButton"
+        style="@style/textButton"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="16dp"
+        android:text="BACK"
+        app:backgroundTint="@color/negative"
+        app:layout_constraintBottom_toTopOf="@id/recipe_display_nav_bar"
+        app:layout_constraintEnd_toStartOf="@id/recipeEditButton"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/RecipeContent" />
```

Comment:
Don't need this if we have top bar

Reply:
As a quick fix, I'll change the button text to "CANCEL". I feel like if i play around with this file, I'll mess up what Boyuan or Elena did.